### DISCUSSION TITLE
Username mentioned notification is triggered without checking word boundaries

### DISCUSSION THREAD

**Discussion by rednir** [ppy/osu#15692] [Q&A] [answered]
I don't know whether this is intentional or not/matches what stable does, but it seems to be causing some confusion:
![image](https://user-images.githubusercontent.com/10225220/142723118-ad4cd580-bfc0-4f04-ad2a-a0b62e58375c.png)

**Comment by bdach** [comment, marked as answer]
escalated to issue #15693
